FinSearchUnified/Tests/Bundle/SearchBundleFindologic/FacetHandler/TextFacetHandlerTest.php

```diff
@@ -0,0 +1,339 @@
+                            false
+                        ),
+                        new ValueListItem(
+                            'FINDOLOGIC',
+                            'FINDOLOGIC',
+                            true
+                        )
+                    ],
+                    'vendor'
+                ),
+                ProductAttributeFacet::MODE_VALUE_LIST_RESULT,
+                new ProductAttributeCondition('vendor', ConditionInterface::OPERATOR_EQ, ['FINDOLOGIC']),
+            ],
+            'Facet with filter without condition' => [
+                [
+                    'name' => 'vendor',
+                    'display' => 'Manufacturer',
+                    'select' => 'single',
+                    'type' => 'label',
+                    'items' => [
+                        [
+                            'name' => 'E-Bike',
+                            'frequency' => 42
+                        ],
+                        [
+                            'name' => 'FINDOLOGIC'
+                        ]
+                    ]
+                ],
+                new ValueListFacetResult(
+                    'product_attribute_vendor',
+                    false,
+                    'Manufacturer',
+                    [
+                        new ValueListItem(
+                            'E-Bike',
+                            'E-Bike (42)',
+                            false
+                        ),
+                        new ValueListItem(
+                            'FINDOLOGIC',
+                            'FINDOLOGIC',
+                            false
+                        ),
+                    ],
+                    'vendor'
+                ),
+                ProductAttributeFacet::MODE_VALUE_LIST_RESULT,
+            ]
+        ];
+    }
+
+    public function radioListDataProvider()
+    {
+        return [
+            'Facet with condition and filter' => [
+                [
+                    'name' => 'vendor',
+                    'display' => 'Manufacturer',
+                    'select' => 'single',
+                    'type' => 'label',
+                    'items' => [
+                        [
+                            'name' => 'FINDOLOGIC',
+                            'frequency' => 42
+                        ]
+                    ]
+                ],
+                new RadioFacetResult(
+                    'product_attribute_vendor',
+                    true,
+                    'Manufacturer',
+                    [
+                        new ValueListItem(
+                            'FINDOLOGIC',
+                            'FINDOLOGIC',
+                            true
+                        )
+                    ],
+                    'vendor'
+                ),
+                ProductAttributeFacet::MODE_RADIO_LIST_RESULT,
+                new ProductAttributeCondition('vendor', ConditionInterface::OPERATOR_EQ, ['FINDOLOGIC']),
```

Comment:
My comment regarding the value array was only meant for `valueListDataProvider`. In this data provider we can keep the single string value since Shopware will also provide us with a single value for radio lists.